### DISCUSSION TITLE
Edge marketplace does not accept new version of my extension

### DISCUSSION THREAD

**Discussion by NabokD** [microsoft/MicrosoftEdge-Extensions#78] [Q&A] [answered]
Could you please clarify what is going wrong and what I should do?
--
Critical validations   1 failure(s)
These failures block the publish of your product to our marketplace.

Requirement Policies

Product Policies
1.8.2 Financial Transactions
The extension description does not provide information about the type of in-product purchases offered.

**Comment by sivsouvamMSFT** [comment]
Dear @NabokD,

As per our content policy, if you are offering in-product purchases on your extension, you must include this information in the product description. You can follow the below path to reach the description and add the details about the in-product offerings. 

`Microsoft Partner Center > Home > Edge >Overview > Your Extension Name > Store Listings > Description`

We appreciate your cooperation and understanding.

Thank you.

**Comment by NabokD** [comment]
It's not working. I've already added it to the description, and it was twice again rejected:
Snipo is available for free on YouTube
The premium plan is $4/month or $36/year for access notes on sites like Udemy and Linkedin Learning

Maybe some other format, or what should I do?

**Comment by NabokD** [comment, marked as answer]
Or I added it to English (United States), but it was not copied to English (United Kingdom) and English, maybe it's a reason, however, it's not obviously enough

**Reply by sivsouvamMSFT** [reply]
Hey @NabokD, could you please share your extension name?

**Reply by NabokD** [reply]
https://microsoftedge.microsoft.com/addons/detail/video-notes-from-youtube-/agjpedhffklaicinofdjnaigdgehipea

**Reply by sivsouvamMSFT** [reply]
@NabokD, we investigated it and found you haven't mentioned details about the premium plan in "English". 
Could you please add the same information to the "English" description same as you did with the English (United States).

**Reply by NabokD** [reply]
Thank you so much. I've added it!

**Reply by NabokD** [reply]
The new version is available, thanks, Siv!
